### PR TITLE
Load datasource configurations

### PR DESCRIPTION
**Pull request by SeverinS**
- Added load configuration from the properties file.
- Created Hikari datasource

Note: This PR should be merged after the PR #3

### CODE REVIEW

**Review by syevtushok — APPROVED** [review]

**Review by syevtushok — APPROVED** [review]

File: src/main/java/com/breskul/bibernate/exeptions/PersistencePropertiesException.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.breskul.bibernate.exeptions;
+
+/**
+ * Throws when the 'persistence.properties' file is not found or can't be loaded
+ *
+ */
+public class PersistencePropertiesException extends CommonException {
+
+    private static final String CANT_FIND_CAUSE = "Unable to find '%s' file";
```
Comment (Mikhail-Mikhailenko) [inline src/main/java/com/breskul/bibernate/exeptions/PersistencePropertiesException.java]: Inside code we should use full words **CAN_NOT**

File: src/main/java/com/breskul/bibernate/exeptions/PersistencePropertiesException.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    private static final String CANT_FIND_SUGGESTION = "The 'persistence.properties' file should be in the classpath";
+
```
Comment (Mikhail-Mikhailenko) [inline src/main/java/com/breskul/bibernate/exeptions/PersistencePropertiesException.java:11]: 'persistence.properties' I think we should accept name file how param. CANT_LOAD_SUGGESTION have the same problem. Using param we avoid dublicate name for file

File: src/main/java/com/breskul/bibernate/exeptions/PersistencePropertiesException.java (hunk 1 above)
Comment (SeverinS) [reply src/main/java/com/breskul/bibernate/exeptions/PersistencePropertiesException.java]: Done

File: src/main/java/com/breskul/bibernate/exeptions/PersistencePropertiesException.java (hunk 2 above)
Comment (SeverinS) [reply src/main/java/com/breskul/bibernate/exeptions/PersistencePropertiesException.java:11]: Done

**Review by Mikhail-Mikhailenko — APPROVED** [review]

**Review by yuriyfomin17 — APPROVED** [review]

**Pull request merged**